packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx

```diff
@@ -94,12 +100,15 @@ function createSignals(signalsConfig: SignalsV1, validate: boolean): Signals {
   const signals: Signals = {}
   if (signalsConfig === undefined) return signals
   Object.entries(signalsConfig).forEach(([name, val]) => {
-    const signal = SIGNALS[val]?.signal
-    if (validate && signal === undefined) {
-      const err = `Signal type ${String(
-        val,
-      )} is unknown. Please check value of ${String(name)} Signal`
-      throw new Error(err)
+    let signal: SignalsZodType | undefined
+    if (typeof val === 'string') {
+      signal = SIGNALS[val]?.signal
```

Comment:
Slight preference to move this after the if on the next line, make this if say "validate && !val in SIGNALS" and then the "let signal" above won't need the "| undefined".